### PR TITLE
#684 - Clean up and reorganized Home.jsx, #685 - Added Twitter error handler and display error message

### LINKED ISSUE #684: ✨ [Feature request]: clean up code home page with reorganization
**Is your feature request related to a problem? Please describe.**
Noticed the `Home.jsx` was getting cluttered and need some tender love and care on organization.

**Describe the solution you'd like**
Reorganize code into separate files.

**Describe alternatives you've considered**
Leave it alone?

### LINKED ISSUE #685: 🐛 [Bug report]: Infinite Loading UI for Twitters section
**Describe the bug**
When twitter widget fails to load their script, the cards won't show and the section is stuck in the loading state.

**To Reproduce**
1. Go to /
2. Once in a blue moon twitter widgets will fail to load
3. Scroll down to Community/Twitter Section 
4. See infinite loading state

**Expected behavior**
Show error or hide section

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Firefox, Chrome

**Additional context**
May not always fire due to external script being fetched

### PR DESCRIPTION
**Pull request by billy-le**
# Description

#684  - moved sections into their own React component to reduce cognitive overload. #685  - added an error handler for when a widget fails to load or when all the widgets fails to load

Fixes #684
Fixes #685

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Only tested locally

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/common/home/sections/banner.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { lazy, Suspense } from "react";
+import "./banner.css";
+
+const LazyBanner = lazy(() =>
```
Comment (harshsinghatz) [inline src/common/home/sections/banner.jsx:4]: wouldn't this always import HackathonBanner
Remove true or replace it with a variable

File: src/common/home/sections/banner.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  true
+    ? import("common/activities/hackathon/HackathonBanner")
+    : import("common/defaultBanner/DefaultBanner")
+);
+
+export default function Banner() {
+  return (
+    <Suspense fallback={<div className="home_banner_fallback" />}>
```
Comment (harshsinghatz) [inline src/common/home/sections/banner.jsx:12]: any specific reason for writing className in this format
everywhere it's written in kebab case, keep it consistent

File: src/common/home/sections/features.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import { RiSlideshow4Line } from "react-icons/ri";
+import { BiShareAlt, BiAddToQueue } from "react-icons/bi";
+import { FaLightbulb } from "react-icons/fa";
+import './features.css';
+
+export default function SectionFeatures() {
+  return (
+    <section className="home-features">
+      <ul className="list-home-features">
+        <li className="home-features-item">
```
Comment (harshsinghatz) [inline src/common/home/sections/features.jsx:11]: You can also make an array of object and loop through them for avoiding repetitive code and also easier to add features later

File: src/common/home/sections/banner.jsx (hunk 1 above)
Comment (billy-le) [reply src/common/home/sections/banner.jsx:4]: oh yes, you're right. I forgot to revert this when i was testing it. will resolve all your other comments. thanks.

File: src/common/home/sections/features.jsx (hunk 3 above)
Comment (billy-le) [reply src/common/home/sections/features.jsx]: hey @harshsinghatz , I've updated the PR with the change requests. let me know if there's anything else I can do to help.

**Comment by billy-le** [comment]
@atapas @harshsinghatz @koustov hello, I can have someone review these changes?

**Comment by harshsinghatz** [comment]
@atapas Please review it!

**Comment by koustov** [comment]
@atapas  are you on it?

**Comment by koustov** [comment]
@billy-le could you please resolve the conflict? I will review post that

**Comment by billy-le** [comment]
@koustov okay, it should be updated now

**Comment by koustov** [comment]
@billy-le thanks. This seems to be a large change. @atapas I feel its better to engage a QA on this PR before moving it next phase

**Pull request closed without merging**